### PR TITLE
adding jobs for openshift/projected-resource-csi-driver

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by coreydaley** [comment]
/assign @adambkaplan @gabemontero 

The projected-resource-csi-driver needs to have a "make verify" .

**Comment by coreydaley** [comment]
I don't think that any of this is going to pass until https://github.com/openshift/projected-resource-csi-driver/pull/1 merges

**Comment by coreydaley** [comment]
@adambkaplan @gabemontero I guess we merge this and do a sample pr to see if it works? :)

File: ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  commands: |
+    mkdir -p /tmp/artifacts
+    make verify > /tmp/artifacts/verify.log
+  container:
+    from: bin
+- artifact_dir: /tmp/artifacts
+  as: unit
+  commands: |
+    mkdir -p /tmp/artifacts/coverage
+    make test OUTPUT_COVERAGE=/tmp/artifacts/coverage > /tmp/artifacts/test-unit.log
```
Comment (gabemontero) [inline ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml:24]: perhaps we should follow this pattern instead:  perhaps we should follow this pattern instead:  https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-openshift-controller-manager-operator/openshift-cluster-openshift-controller-manager-operator-master.yaml#L44-L45

I think the output showing up the in logs is sufficient capture  vs. storing them in a file, no?

File: ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  '*':
+    requests:
+      cpu: "3"
+      memory: 1Gi
+tests:
+- artifact_dir: /tmp/artifacts
+  as: verify
+  commands: |
+    mkdir -p /tmp/artifacts
+    make verify > /tmp/artifacts/verify.log
```
Comment (gabemontero) [inline ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml:17]: don't think we should redirect this to a file here ... just run `make verify`

File: ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml (hunk 1 above)
Comment (coreydaley) [reply ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml]: updated in current push

File: ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml (hunk 2 above)
Comment (coreydaley) [reply ci-operator/config/openshift/projected-resource-csi-driver/openshift-projected-resource-csi-driver-master.yaml]: updated in current push

**Comment by coreydaley** [comment]
@gabemontero @adambkaplan all green again

**Comment by gabemontero** [comment]
@adambkaplan needs to approve / lgtm (he is the one in the parent OWNERS file)

but I'm LGTM fwiw - thanks @coreydaley

**Pull request merged**